src/Server.UI/Pages/Participants/MovedParticipants.razor

```diff
@@ -0,0 +1,64 @@
+﻿@page "/pages/participants/movedparticipants"
+
+@using Cfo.Cats.Application.Features.Transfers.DTOs
+@using Cfo.Cats.Application.Features.Transfers.Queries
+@using Cfo.Cats.Application.SecurityConstants
+
+@inherits CatsComponentBase
+
+@attribute [Authorize(Policy = SecurityPolicies.UserHasAdditionalRoles)]
+
+<MudText Typo="Typo.h2">Participants You're Losing Access To'</MudText>
+<MudSimpleTable>
+    <thead>
+        <tr>
+            <th>Participant</th>
+            <th>Occurred</th>
+            <th>From</th>
+            <th>To</th>
+            <th>Type</th>
+        </tr>
+    </thead>
+    <tbody>
+        @foreach (var transfer in outgoingTransfers.OrderByDescending(t => t.MoveOccured))
+        {
+            <tr>
+                <td>@transfer.ParticipantId</td>
+                <td>@transfer.MoveOccured.ToLocalTime().ToString("dd/MM/yyyy HH:mm")</td>
+                <td>@transfer.FromLocation.Name (@transfer.FromLocation.LocationType.Name)</td>
+                <td>@transfer.ToLocation.Name (@transfer.ToLocation.LocationType.Name)</td>
+                <td>@transfer.TransferType.Name</td>
+                <MudMenuItem OnClick="@(()=> Edit(transfer.ParticipantId))">
+                    @ConstantString.Edit
+                </MudMenuItem>
+            </tr>
+        }
+    </tbody>
+</MudSimpleTable>
+
+@code {
+    private IEnumerable<OutgoingTransferDto> outgoingTransfers = [];
+
+    protected async override Task OnInitializedAsync()
+    {
+        outgoingTransfers = await GetOutgoingTransfers();
+        await base.OnInitializedAsync();
+    }
+    
+    private async Task<IEnumerable<OutgoingTransferDto>> GetOutgoingTransfers()
+    {
+        var query = await GetNewMediator().Send(new GetOutgoingTransfers.Query());
```

Comment:
I'm not sure relying on Outgoing Transfers is entirely sufficient here... Yes, this will show people leaving your contract (and therefore everybody you're losing access to), but, the vast majority of these participants a support worker may never have actually worked with, as they were working with another support worker, possibly even in a location they don't have access to.

We need to consider the below in order to make this screen as easy to find a participant as possible:
- Backdating (90 days), with consideration of finding someone here after the 90 days (as per the rules)
- Only showing participants you were working with (assigned to you), but have lost (or losing) access to